website/content/partials/api/restricted-endpoints.mdx

```diff
@@ -10,7 +10,7 @@ API path                                    | Root | Admin
 `sys/audit-hash`                            | YES  | YES
 `sys/config/auditing/*`                     | YES  | NO
 `sys/config/cors`                           | YES  | NO
-`sys/config/group-policy-application`       | YES  | NO
+`sys/config/group-policy-application`       | YES  | YES
```

Comment:
This change is fine, but we also need to update `/api-docs/system/config-group-policy-application.mdx` to use the `alerts/restricted-admin.mdx` partial instead of `alerts/restricted-root.mdx`

Reply:
Thanks! I totally missed this.